projects/storefrontapp-e2e-cypress/cypress/integration/b2b/order-approval/b2b-order-approval.e2e-spec.ts

```diff
@@ -0,0 +1,275 @@
+import { tabbingOrderConfig as config } from '../../../helpers/accessibility/b2b/tabbing-order.config';
+import * as orderApproval from '../../../helpers/b2b/b2b-order-approval';
+import * as sampleData from '../../../sample-data/b2b-order-approval';
+
+describe('B2B - Order Approval', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+  });
+
+  describe('Order Approval - Accessibility.', () => {
+    beforeEach(() => {
```

Comment:
Logging in consumes some seconds. The pattern that we are following in other tests is:
- Log in once
- Store the token in local storage
- restore local storage before each test, to get the token
That way, you don't have to login again. You'll reduce the execution time for at least 30 seconds by logging in once or twice as an approver instead of 8 times (before each test)

```suggestion
    before(() => {
      orderApproval.loginB2bApprover();
      cy.saveLocalStorage();
    });

    beforeEach(() => {
      cy.restoreLocalStorage();
    });
```

Reply:
Done.